File: app/src/main/java/com/lifedawn/bestweather/retrofit/responses/kma/json/ultrasrtfcstresponse/UltraSrtFcstRoot.java
package com.lifedawn.bestweather.retrofit.responses.kma.json.ultrasrtfcstresponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.lifedawn.bestweather.retrofit.responses.kma.json.vilagefcstcommons.VilageFcstResponse;

public class UltraSrtFcstRoot {
	@Expose
	@SerializedName("response")
	private VilageFcstResponse response;
	
	
	public void setResponse(VilageFcstResponse response) {
		this.response = response;
	}
	
	public VilageFcstResponse getResponse() {
		return response;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/ultrasrtfcstresponse/UltraSrtFcstRoot.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.ultrasrtfcstresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.vilagefcstcommons.VilageFcstResponse

class UltraSrtFcstRoot {
    @Expose @SerializedName("response") var response: VilageFcstResponse? = null
}